packages/fx-core/tests/plugins/resource/function/unit/scaffold.test.ts

```diff
@@ -1,133 +1,148 @@
-        afterEach(() => {
-            fs.emptyDirSync(context.root);
-            fs.rmdirSync(context.root);
-            sinon.restore();
-        });
-
-        it("Test pre-scaffold without function name", async () => {
-            // Arrange
-            context.answers = new Map<string, string>();
-            const plugin: FunctionPlugin = new FunctionPlugin();
-
-            // Act
-            const ret: FxResult = await plugin.preScaffold(context);
-
-            // Assert
-            chai.assert.isTrue(ret.isOk());
-        });
-
-        it("Test scaffold", async () => {
-            // Arrange
-            context.answers = new Map<string, string>([
-                [QuestionKey.functionName, "httpTrigger"],
-                [QuestionKey.programmingLanguage, FunctionLanguage.JavaScript],
-            ]);
-            const zip = new AdmZip();
-            zip.addFile("test.js.tpl", Buffer.from("{{appName}} {{functionName}}"));
-            sinon.stub(fetch, "getTemplateURL").resolves(undefined);
-            sinon.stub(fetch, "fetchZipFromURL").resolves(zip);
-
-            const plugin: FunctionPlugin = new FunctionPlugin();
-
-            // Act
-            await plugin.preScaffold(context);
-            const ret: FxResult = await plugin.scaffold(context);
-
-            // Assert
-            chai.assert.isTrue(ret.isOk());
-        });
-
-        it("Test scaffold with additional function", async () => {
-            // Arrange
-            context.answers = new Map<string, string>([
-                [QuestionKey.functionName, "httpTrigger"],
-                [QuestionKey.programmingLanguage, FunctionLanguage.JavaScript],
-            ]);
-            const zip = new AdmZip();
-            zip.addFile("test.js.tpl", Buffer.from("{{appName}} {{functionName}}"));
-            sinon.stub(fetch, "getTemplateURL").resolves(undefined);
-            sinon.stub(fetch, "fetchZipFromURL").resolves(zip);
-
-            const plugin: FunctionPlugin = new FunctionPlugin();
-
-            // Act
-            await plugin.preScaffold(context);
-            const ret: FxResult = await plugin.scaffold(context);
-
-            // Assert
-            chai.assert.isTrue(ret.isOk());
-        });
-
-        it("Test scaffold with fallback in JS", async () => {
-            // Arrange
-            context.answers = new Map<string, string>([
-                [QuestionKey.functionName, "httpTrigger"],
-                [QuestionKey.programmingLanguage, FunctionLanguage.JavaScript],
-            ]);
-            sinon.stub(fetch, "getTemplateURL").rejects(new Error());
-            const plugin: FunctionPlugin = new FunctionPlugin();
-
-            // Act
-            await plugin.preScaffold(context);
-            const ret: FxResult = await plugin.scaffold(context);
-
-            // Assert
-            chai.assert.isTrue(ret.isOk());
-        });
-
-        it("Test scaffold with fallback in TS", async () => {
```

Comment:
Remove the ts version test because the templates file is mock, so language does not matter o n this test any more.